renku/service/utils/__init__.py

```diff
@@ -74,3 +84,13 @@ def repo_sync(repo_path, remote_names=('origin', )):
                 repo.git.push(remote.name, repo.active_branch)
 
     return pushed_branch
+
+
+def new_repo_push(
+    repo_path, source_url, source_name='origin', source_branch='master'
```

Comment:
If https://github.com/SwissDataScienceCenter/renku-python/pull/1273 gets merged first, this should be updated to also use the `renku.core.commands.save.repo_sync` introduced there

Reply:
On nice. In that case, I should still set up a remote for a new project, right? Maybe that step could still go in this function where I invoke `repo_sync`

Reply:
Yes that makes sense 👍  (Sorry for the late reply...)

Reply:
oh I realize only now that I haven't addressed this :facepalm: 
I'll give it a look right now